pkgs/tools/text/par/default.nix

```diff
@@ -18,7 +18,11 @@ stdenv.mkDerivation {
     })
   ];
 
-  buildPhase = ''make -f protoMakefile'';
+  buildPhase = ''
+    make -f protoMakefile \
+      CC="${stdenv.cc.targetPrefix}cc -c" \
+      LINK1=${stdenv.cc.targetPrefix}cc
+  '';
```

Comment:
I did not saw this earlier, but should it not be feasible to just use `makeFlags` here?
 
makeFlags = [ 
  "-f" "protoMakefile"
  "CC=${stdenv.cc.targetPrefix}cc -c"
  "LINK1=${stdenv.cc.targetPrefix}cc"
];

Reply:
Does not work for me due space inside of CC. -c gets interpreted as make(1) flag.

Reply:
But can work with makeFlagsArray. Do you like this approach (just pushed) better?